src/pull-requests/dtos/pull-request-contributor-options.dto.ts

```diff
@@ -27,7 +27,7 @@ export class PullRequestContributorOptionsDto extends PageOptionsDto {
   })
   @IsString()
   @IsOptional()
-  readonly repo?: string;
+  readonly repos?: string;
```

Comment:
Is this a delimited string of repos now or should this still be

```suggestion
  readonly repo?: string;
```

Reply:
Exactly: this can now accept a comma delimited list of repos (like in other places)

https://github.com/open-sauced/api/pull/490/files#diff-07bc12f335db01dfed369b3b57eeabb2014e16abf8a22fc3776bbfee06a539f0R532-R537

I'll update the description to denote that